packages/terra-embedded-content-consumer/docs/README.md

```diff
@@ -0,0 +1,30 @@
+# Terra Embedded Content Consumer
+
+The Embedded Content Consumer is the application component which is embedding content within it.
+
```

Comment:
Your PR description is: 
> An Embedded content consumer component to interface with embedded iframe content

I feel it would be good to note this is meant specifically for iframe content.

Reply:
How about "embedding web content" instead of calling out the internal iframe details?

Reply:
Can you make this change in all description places?